appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -419,8 +419,10 @@ private YaVersion() {
   // For YOUNG_ANDROID_VERSION 166:
   // - MAP_COMPONENT_VERSION was incremented to 3
   // - FEATURE_COLLECTION_COMPONENT_VERSION was incremented to 2
+  // For YOUNG_ANDROID_VERSION 167:
+  // - IMAGE_COMPONENT_VERSION was incremented to 4
 
-  public static final int YOUNG_ANDROID_VERSION = 166;
+  public static final int YOUNG_ANDROID_VERSION = 167;
```

Comment:
This isn't a hard policy yet, but we are thinking that `YOUNG_ANDROID_VERSION` should only be updated when we officially publish to a server, similarly to how we handle the change to the Companion version. This is okay for now (just a heads up).